packages/@aws-cdk/aws-apigateway/lib/json-schema.ts

```diff
@@ -0,0 +1,125 @@
+export enum JsonSchemaVersion {
```

Comment:
Can we bring this from a dependency instead of implementing ourselves?

Reply:
Most dependencies have all the logic for validation, here we only have 1 interface and a simple replacement function for the JSON generation that puts the '$' on front of type, id, and ref when serializing.